launch/docker_test.go

```diff
@@ -303,5 +495,25 @@ func TestHelperProcess(t *testing.T) {
 			os.Exit(1)
 		}
 		os.Exit(0)
+	case "SUCCESS_TO_KILL":
+		if subcmd == "command" {
+			time.Sleep(fakeProcessLifeTime)
+			os.Exit(0)
+		}
+		os.Exit(0)
+	case "SUCCESS_TO_KILL_DOCKER_CMD":
```

Comment:
This process is same with "SUCCESS_TO_KILL". You can use one of them.

Reply:
thx.
`SUCCESS_TO_KILL_DOCKER_CMD` has been merged into `SUCCESS_TO_KILL`.